package.json

```diff
@@ -63,6 +63,6 @@
     "typescript": "~5.6.2",
     "typescript-eslint": "^8.15.0",
     "vite": "^6.0.1",
-    "vitest": "^2.1.8"
```

Comment:
I bumped these to see if it would just fix the intermittently failing tests the release note's breaking changes didn't seem applicable to us but, this didn't make much of a difference I figure we should keep up to date but, if people want I can just bump them back down